frontend/recipe/nonselectablecomboboxitems/vaadin-combo-box-item-styles.css

```diff
@@ -0,0 +1,3 @@
+:host([theme~="non-selectable-items"]) {
+	pointer-events: none;
```

Comment:
This prevents mouse selection, can we prevent keyboard selection as easily?

Reply:
Not possible to do through CSS it seems: https://stackoverflow.com/a/14072495/11982497, 
and I couldn't figure out a JS approach (not one that is simple at least). 

To my mind, this requires a not-so-simple refactoring of `_onArrowDown()` and `_onArrowUp()` in [vaadin-combo-box-mixin](https://github.com/vaadin/vaadin-combo-box/blob/master/src/vaadin-combo-box-mixin.html) to skip disabled items.

Reply:
But this is a fatal flaw IMO. So, unless someone has a good idea for a simple fix for disabling keyboard selection, perhaps this is not a viable recipe afterall.